<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RadiationReconnaissanceResultsDockWidgetBase</class>
 <widget class="QDockWidget" name="RadiationReconnaissanceResultsDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>231</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>269</width>
    <height>222</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Radiation reconnaissance results</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="main">
       <attribute name="title">
        <string>Main</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="raster_label">
          <property name="text">
           <string>Raster:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QgsMapLayerComboBox" name="raster_box"/>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="load_raster">
          <property name="text">
           <string>Load raster</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="type_label">
          <property name="text">
           <string>Type of data:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="type_box">
          <item>
           <property name="text">
            <string>Activity</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Doserate</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="report_label">
          <property name="text">
           <string>Report:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="report_file">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QToolButton" name="report_button">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="solve_button">
          <property name="text">
           <string>Solve</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settings">
       <property name="accessibleName">
        <string/>
       </property>
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="8" column="0" colspan="2">
         <widget class="QCheckBox" name="check_shp">
          <property name="text">
           <string>Create shapefile of polygons</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="check_100">
          <property name="text">
           <string>100</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="doserate_label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="check_50">
          <property name="text">
           <string>50</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QCheckBox" name="check_1000">
          <property name="text">
           <string>1000</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="check_10">
          <property name="text">
           <string>10</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="check_5">
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="check_1">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="check_01">
          <property name="text">
           <string>0.1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="check_001">
          <property name="text">
           <string>0.01</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="2">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgis.gui</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
